flixel/FlxObject.hx

```diff
@@ -946,6 +947,47 @@ class FlxObject extends FlxBasic
 	}
 	
 	/**
+	 * Aligns this FlxObject to a target by an anchor point.
+	 * 
+	 * @param	target	To target object to be aligned with.
+	 * @param	anchor	On what anchor point to align the object - default is CENTER.
+	 * @return 	This FlxObject for chaining
+	 */
+	public function alignTo(target:FlxObject, ?anchor:FlxAlign):FlxObject
+	{
+		if (anchor == null)
```

Comment:
Like I mentioned, I don't really like the API as is, moving it to a static extension wouldn't that. flixel-ui's approach seems much more flexible.